aten/src/ATen/core/Tensor.h

```diff
@@ -586,6 +586,7 @@ class CAFFE2_API Tensor {
   Scalar q_scale() const;
   Scalar q_zero_point() const;
   Tensor int_repr() const;
+  int64_t qscheme() const;
```

Comment:
is this the API we actually want for qscheme?  I would assume we'd want the actual thing at the C++ level, and in python you could either do the string or int thing.

Or if you want to actually expose this to python that could be done via the same method as memory_format, see https://github.com/pytorch/pytorch/pull/20455.

Reply:
this is because native_function only supports int, looks like memory_format is something we want, I'll follow that pr.